src/services/tweets.service.ts

```diff
@@ -42,7 +53,21 @@ export const updateTweet = (
     updatedAt: string;
   }
 ) => {
-  return updateOne(id, newData);
+  const tweetOldData = await getOneById(id);
+  const updateResult = await updateOne(id, newData);
+
+  if (updateResult) {
+    const oldHashtags = TweeterUtils.extractHashtags(
+      convert(tweetOldData.content)
+    );
+    const newHashtags = TweeterUtils.extractHashtags(convert(newData.content));
+    const hashtagsDiff = newHashtags.filter((x) => !oldHashtags.includes(x));
```

Comment:
create normal name of variable.
What if we had some tags before but now delete it?
Check this situation too use function decrementStatsByName